ddl/generated_column.go

```diff
@@ -65,6 +84,34 @@ func checkDependedColExist(dependCols map[string]struct{}, cols []*table.Column)
 	return nil
 }
 
+// findPositionRelativeColumn return a pos relative to added generated column position
+func findPositionRelativeColumn(cols []*table.Column, pos *ast.ColumnPosition) (int, error) {
+	position := len(cols)
+	// get column position default is append behind
```

Comment:
```suggestion
	// get column position default is append behind
```
What does this comment mean? I'm confused

Reply:
Addressed. 
It's should be: Get column position, the default is cols's length means appending behind.